cmd/app/list/cmd.go

```diff
@@ -35,5 +43,5 @@ func run(cmd *cobra.Command) error {
 
 func init() {
 	flags := Cmd.Flags()
-	args.AddOrganizationSlugFlag(flags, "to list apps from", &cmdArgs.organizationSlug)
+	flags.StringVarP(&cmdArgs.organizationSlug, "organization", "o", "", "The organization slug identifier to list apps from. List available organizations with 'numerous organization list'.")
```

Comment:
Why did you reverse this refactor? 😄 Just curious

Reply:
It gave the wrong message for the app list command!

Reply:
Because that functionality says `"The organization slug identifier of the app"`, and this does not relate to a specific app.

Reply:
What did it say?

Reply:
The app list command is not related to a specific app, but the helper methods assumed that.

So the message had the format "The organization slug identifier of the app <ACTION TEXT>". And the resulting message was "The organization slug identifier of the _app_ to list apps from"